safe-global/web-core
# feat: `msgsSlice` store slice

## What it solves

Partially resolves #1159

## How this PR fixes it

A new `msgsSlice` store slice has been created and associated hooks created/updated: `useLoadMessages`, `useMessages`.

## How to test it

Open the "Messages" in the transaction list and observe that two mock messges are there.
